server/src/main/java/org/elasticsearch/monitor/metrics/NodeMetrics.java

```diff
@@ -359,7 +359,7 @@ private void registerAsyncMetrics(MeterRegistry registry) {
 
         metrics.add(
             registry.registerLongGauge(
-                "es.node.stats.indices.indexing.primary_operations.memory.size.current",
+                "es.node.stats.indices.indexing.primary_operations.memory.size.total",
```

Comment:
Same as above, it is probably just `size`?

Reply:
agree